src/test/java/org/takes/facets/auth/TokenTest.java

```diff
@@ -67,7 +67,9 @@ public void joseEncoded() throws IOException {
         final JsonObject jose = new Token.Jose(256).json();
         MatcherAssert.assertThat(
             code,
-            Matchers.equalTo(new Base64().encode(jose.toString()))
+            Matchers.equalTo(
+                Base64.getEncoder().encode(jose.toString().getBytes())
```

Comment:
@paulodamaso I think it's better to test actual value against contstant string instead of dynamic function result, in such way we can handle cases where function implementation is different for some jdk implementations, e.g. it may return `null` in one jdk for invalid input and throws exception in another.